extensions/ql-vscode/package.json

```diff
@@ -1732,6 +1732,7 @@
     "format": "prettier --write **/*.{ts,tsx} && eslint . --ext .ts,.tsx --fix",
     "lint": "eslint . --ext .js,.ts,.tsx --max-warnings=0",
     "lint:markdown": "markdownlint-cli2 \"../../**/*.{md,mdx}\" \"!**/node_modules/**\" \"!**/.vscode-test/**\" \"!**/build/cli/v*/**\"",
+    "find-deadcode": "ts-unused-exports find-deadcode-tsconfig.json --showLineNumber --excludePathsFromReport=$(find src test -type f '(' -name jest.config.ts -or -name 'index.tsx' -or -name 'index.ts' ')' -print0 | tr '\\0' ';')'gulpfile.ts/;src/stories/;test/vscode-tests/jest-runner-installed-extensions.ts'",
```

Comment:
There are quite a few files that either have to have unused exported items by design (e.g. the `index.tsx` files or anything in `src/stories/`), and perhaps that we just don't care as much about so it's easier to ignore them to avoid jumping through hoops. Feel free to say if you think this list is not correct.